packages/components/src/styles/components/app-side-nav/toggle-button.scss

```diff
@@ -10,13 +10,13 @@
 
 .hds-app-side-nav__toggle-button {
   position: absolute;
-  top: 22px; // visually align with :header content
+  top: var(--token-side-nav-wrapper-padding-vertical); // visually align with :header content
```

Comment:
[issue?] the token is `token-side-nav` but the component is `app-side-nav`: should the token be specific to the **App**SideNav component (also considering that the SideNav will be deprecated and likely removed?

Reply:
Yes, I probably just made a mistake. I will check.

Reply:
Updated